docs/analysis_software_MRI.md

```diff
@@ -1,48 +1,97 @@
-* [course/tutorial](http://www.fil.ion.ucl.ac.uk/spm/course/)
-
-#### Freesurfer
-* [website](https://surfer.nmr.mgh.harvard.edu/)
-* [manual](https://surfer.nmr.mgh.harvard.edu/fswiki)
-* [mailing list](https://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferSupport)
-* [course](https://www.youtube.com/channel/UCruQerP8aa-gYttXkAcyveA) and [videos](https://surfer.nmr.mgh.harvard.edu/fswiki/CourseDescription)
-
-#### FSL
-* [website](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki)
-* [manual](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Support)
-* [mailing list](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=fsl)
-* [course/tutorial](https://fsl.fmrib.ox.ac.uk/fslcourse/)
-
-#### AFNI
-* [website](https://afni.nimh.nih.gov/)
-* [manual](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/index.html)
-* [mailing list](https://afni.nimh.nih.gov/afni/community/board/list.php?1)
-* course/tutorial: Video recordings from the [AFNI bootcamp](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/educational/bootcamp_recordings.html), with slides, and [example data](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/background_install/unix_tutorial/misc/install.data.html).
-
-#### ANTs
-* [website](http://stnava.github.io/ANTs/)
-* [manual/wiki](https://github.com/ANTsX/ANTs/wiki)
-
-#### Nipype
-Nipype is best viewed as a way to create and run software-agnostic preprocessing/analysis-pipeline. It becomes very powerful when you need to use different softwares in your analysis.
-* [website](https://nipype.readthedocs.io/en/latest/)
-* [course/tutorial](https://miykael.github.io/nipype_tutorial/)
-
-#### Dipy
-Diffusion neuroimaging in Python
-* [website](https://github.com/ohbm/hackathon2019)
-
-#### MRtrix3
-Advanced tools for the analysis of diffusion MRI data
-* [website](http://www.mrtrix.org/)
-* [community forum](http://community.mrtrix.org/)
-* [user documentation](https://mrtrix.readthedocs.io/en/latest/)
-* [developer documentation](http://www.mrtrix.org/developer-documentation/)
-* [manuscript preprint](https://www.biorxiv.org/content/10.1101/551739v1). Includes example code for both C++11 and Python libraries.
-* [BATMAN tutorial](https://osf.io/fkyht/). MRtrix3 tutorial available on OSF with an awesome title: B.A.T.M.A.N., the Basic and Advanced Tractography with MRtrix for All Neurophiles
+# Neuroimaging analysis software
+
+
+- a software: [AFNI](https://afni.nimh.nih.gov/)
+    - description: A software suite primarily developed for the analysis and display of anatomical and fMRI data.
+    - repository URL:
+    - website URL: [https://afni.nimh.nih.gov/](https://afni.nimh.nih.gov/)
+    - tutorial URL: [tutorials](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/tutorials/main_toc.html), [educational examples](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/educational/main_toc.html)
+    - documentation: [https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/index.html](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/index.html)
```

Comment:
Same here?

```
- [documentation](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/index.html)
```

Reply:
Also agreed! Neater this way. I was thinking about it myself. I can change the learn_neuroimaging.md page later to implement these changes there as well.

Reply:
OK @IRuotsa let me know what you think.

I think that you are OK I will merge this and then we can sort come up with a good enough template at our meeting and then come back to implement those changes

Reply:
I was also unsure of these, but thought to raise these issues on today's meeting. But if you can merge these now. I'll fix them later. Thanks!

Reply:
OK merging.